ehitchhiking-ui/src/app/pages/registration/registration.component.ts

```diff
@@ -19,4 +28,8 @@ export class RegistrationComponent implements OnInit {
 			this.router.navigateByUrl('/main');
 		}
 	}
+
+	hasError(controlName: string, errorName: string): boolean {
+		return this.registrationForm.controls[controlName].hasError(errorName);
```

Comment:
return this.registrationForm.controls && this.registrationForm.controls[controlName] && this.registrationForm.controls[controlName].hasError(errorName)

Reply:
fixed